### ISSUE TITLE
[A11y] Issues noticed in a11y review

### ISSUE THREAD

**Issue by mbmjertan** [infinum/eightshift-frontend-libs#596]
- [ ] Add warning when autoplay and sound are enabled, and video controls are disabled
- [ ] Jumbotron content can overflow on small resolutions
- [ ] Jumbotron can be a single section, not a section and a graphic
- [ ] Links in navigation should be distinguished with underlines
- [ ] List type controls in editor are not working
